src/app/tests/suites/tests.js

```diff
@@ -216,8 +212,6 @@ function getManualTests() {
     'Test_TC_MF_1_24',
     'Test_TC_MF_1_25',
     'Test_TC_MF_1_26',
-    'Test_TC_MF_1_27',
```

Comment:
Is there a reason why these are being removed? There were just added in this [PR](https://github.com/project-chip/connectedhomeip/pull/17837)

Reply:
Might be a case of TC name being renamed ? @kowsisoundhar12 can you confirm ?

Reply:
@raju-apple @krypton36 recently i updated this scripts based on latest  xl sheet but Manjunath PR is conflict with this PR so i removed this manual script

https://docs.google.com/spreadsheets/d/1h4d4J90KAxFqUL_cXZHyM3pHjHCEXzFITkQhN2KMqDY/edit#gid=710954404